Scripts/Helpers/ProxyNode.cs

```diff
@@ -554,14 +632,21 @@ void ExecuteFeedback(ProxyFeedbackRequest changedRequest)
                         if (tooltip)
                         {
                             tooltip.tooltipText = tooltipText;
-                            this.ShowTooltip(tooltip, true, placement: tooltip.GetPlacement(m_FacingDirection));
+                            this.ShowTooltip(tooltip, true, placement: tooltip.GetPlacement(m_FacingDirection),
+                                becameVisible: () =>
```

Comment:
Let's check how much allocs this is giving us (vs. having some function inside data itself that can be passed as the becameVisible action?  I think that will still alloc but not as much).  I don't think feedback happens so much that this will be an issue, just something to be aware of.

Reply:
I've been thinking of feedback as "not per-frame" for the time being, but seeing how often we change feedback, we should probably start worrying about GC. I made a similar comment in the other PR about cacheing FeedbackRequests, too.

I'll kill any GC here if it's happening.